src/main/webapp/app/shared/subject/subject-pair-dialog.component.html

```diff
@@ -53,20 +62,5 @@ <h4 class="modal-title" id="mySubjectLabel" jhiTranslate="managementPortalApp.su
         <button type="button" class="btn btn-default" data-dismiss="modal" (click)="clear()">
             <span class="fa fa-ban"></span>&nbsp;<span jhiTranslate="entity.action.close">Close</span>
         </button>
-        <button type="button" class="btn btn-warning"
-                *ngIf="showTokenUrl"
-                [disabled]="!showQRCode"
-                (click)="lockTokenUrl()">
-            <span class="fa fa-lock"></span>&nbsp;<span jhiTranslate="managementPortalApp.subject.hideTokenUrl">Hide Token Url</span>
-        </button>
-        <button type="button" class="btn btn-warning"
-                *ngIf="!showTokenUrl"
-                [disabled]="!showQRCode"
-                (click)="unlockTokenUrl()">
-            <span class="fa fa-unlock"></span>&nbsp;<span jhiTranslate="managementPortalApp.subject.showTokenUrl">Show Token Url</span>
-        </button>
-        <button type="button" class="btn btn-primary" [disabled]="!showQRCode" (click)="updateQRCode()">
```

Comment:
The non persistent tokens can still be fetched only once. Wouldn't we want to have option to refresh if we choose to generate QR code? (not the persistent one)

Reply:
The Generate QR code button remains active, and can be pressed again to regenerate.